server/src/pages/shot/view.js

```diff
@@ -80,6 +78,16 @@ class Clip extends React.Component {
     // Allow default action to continue
   }
 
+  textCaptureEnabled() {
```

Comment:
`textCaptureEnabled` sounds like a true/false function. `textCaptureIfEnabled` would be a little clearer, but I would just name this `copyTextContextMenu()`.

Also it should return null at the end instead of the implicit return undefined.